lib/pkgcloud/openstack/orchestration/stack.js

```diff
@@ -45,15 +45,12 @@ Stack.prototype._setProperties = function (details) {
   this.createdAt = details['creation_time'];
   this.updatedAt = details['updated_time'];
 
+  this.outputs = details['outputs'];
```

Comment:
Perfect, thanks. So it doesn't appear to come back from `createStack` but rather `getStack`?